### PR TITLE
Added CUTLASS to the list of GEMM kernels.

### PR DESCRIPTION
**Pull request by adityaiitb**
The new docker image includes GEMM kernels from CUTLASS.

Signed-off-by: Aditya Agrawal <aditya.iitb@gmail.com>

### CODE REVIEW

File: pyprof/prof/linear.py [hunk 1]
```diff
@@ -107,7 +112,7 @@ def __init__(self, d):
         if any(x in d.name for x in Linear.gemmKernels):
             self.op_ = "linear"
         else:
-            assert any(x in d.name for x in Linear.biasKernels), f"Kernel data: {d}"
+            assert any(x in d.name for x in Linear.biasKernels), "Kernel Name: {}".format(d.name)
```
Comment (debermudez) [inline pyprof/prof/linear.py:115]: Why change this back to .format syntax. According to this site, https://realpython.com/python-f-strings/, f-strings are a more recent pythonic way of programming string output.

File: pyprof/prof/linear.py [hunk 2]
```diff
@@ -29,10 +29,15 @@ class Linear(OperatorLayerBase):
 	bias in bprop is a reduction and hence is 1 read.
 	'''
 
-    gemmKernels = ["gemm", "gemv", "dot_kernel", "splitKreduce_kernel", "reduce_1Block_kernel"]
+    gemmKernels = [
+        "gemm", "gemv", "dot_kernel", "splitKreduce_kernel",
+        "reduce_1Block_kernel", "cutlass"
```
Comment (debermudez) [inline pyprof/prof/linear.py:34]: What prompted this addition? Did a user get an assertion?

Comment (adityaiitb) [reply pyprof/prof/linear.py:34]: I got an assertion while running NCF with the new docker, which presumably has a newer cuda tool kit / cublas.

File: pyprof/prof/linear.py (hunk 1 above)
Comment (adityaiitb) [reply pyprof/prof/linear.py]: Fixed.

**Pull request merged**